distribution/lib/Standard/Test/0.0.0-dev/src/Bench.enso

```diff
@@ -161,8 +161,10 @@ type Bench
         case get_benchmark_report_path of
             Nothing -> Nothing
             path ->
-                line = 'Label,Phase,"Invocations count","Average time (ms)","Time Stdev"'
-                line.write path on_existing_file=Existing_File_Behavior.Backup
+                f = File.new path
+                if f.exists.not then
+                    line = 'Label,Phase,"Invocations count","Average time (ms)","Time Stdev"'
+                    line.write f on_existing_file=Existing_File_Behavior.Backup
```

Comment:
Should this file existence check also be done in `summarize_phase`? If we want to avoid writing to an existing file, shouldn't it be done in both places?